app/helpers/date_validation_helper.rb

```diff
@@ -3,6 +3,11 @@ def valid_or_invalid_date(year, month)
     raise ArgumentError if year.blank?
 
     date_args = [year, month, 1].map(&:to_i)
+
+    if date_args[1].zero?
+      date_args[1] = Date.parse(month).month
```

Comment:
Clever! :rocket:

Reply:
Haha thanks stranger! Not as clever as your code though 🤯